src/java/org/tensorics/core/lang/ManipulationOptions.java

```diff
@@ -70,14 +70,28 @@ private ManipulationOptions() {
     @SuppressWarnings("deprecation")
     public static <T> OptionRegistry<ManipulationOption> defaultOptions(ExtendedField<T> field) {
         return ImmutableOptionRegistry.of(ImmutableList.of(//
-                new IntersectionShapingStrategy(), //
+                IntersectionShapingStrategy.get(), //

```

Comment:
actually, I think we could think of adding singleton getters or factory methods everywhere ... here this looks a little inconsistent.